### ISSUE TITLE
Tag v0.9.6 appears to be misplaced

### ISSUE THREAD

**Issue by tomzy-0** [Neargye/magic_enum#372]
I think that there is something wrong with latest tag. v0.9.6 is placed on a commit named `v0.9.5`.

```
λ git log --graph --decorate --pretty=oneline --abbrev-commit --all
* dae6bbf16c36 (HEAD -> master, origin/master, origin/HEAD) Bump the github-actions group with 3 updates (#370)
* ec8b0e985f25 Update comments regarding max and min (#369)
* 9a9308fd5477 fix bitset constructor (#367)
* 664ee62c1257 Parameterize formatter::format to use with FMT_COMPILE (#365)
* ec1adc5763f6 Bump github/codeql-action in the github-actions group (#364)
* dd6a39d0ba18 v0.9.6
| * adc7d2ba57e1 (tag: v0.9.6) v0.9.5
|/  
* 0c1d10a62490 fix tests
* f08355cd0d63 fix enum_reflected
* bc2e94853b26 add enum_reflected
* 2ac8152a2b8c update ci (#363)
* f2ad79f06bed fix #362
```

Commits `adc7d2ba57e1` and `dd6a39d0ba18` are the same? Should I use `dd6a39d0ba18` If I want magic_enum in v0.9.6?

**Comment by Neargye** [comment]
Hi,
Now `v0.9.6`  on `dd6a39d0ba18`

Thanks for your issues, I tried to fix it in git. (To be honest, I didn't understand and I don't remember how it happened, I think it was a typo)

**Comment by tomzy-0** [comment]
Thank you for fixing this. Now git history is not misleading :+1:

```
λ git log --graph --decorate --pretty=oneline --abbrev-commit --all
* dae6bbf16c36 (HEAD -> master, origin/master, origin/HEAD) Bump the github-actions group with 3 updates (#370)
* ec8b0e985f25 Update comments regarding max and min (#369)
* 9a9308fd5477 fix bitset constructor (#367)
* 664ee62c1257 Parameterize formatter::format to use with FMT_COMPILE (#365)
* ec1adc5763f6 Bump github/codeql-action in the github-actions group (#364)
* dd6a39d0ba18 (tag: v0.9.6) v0.9.6
* 0c1d10a62490 fix tests
* f08355cd0d63 fix enum_reflected
* bc2e94853b26 add enum_reflected
* 2ac8152a2b8c update ci (#363)
* f2ad79f06bed fix #362
* bf3c74ab7ede Add better support for Yocto/PetaLinux (#360)
```

I will close the issue
